### PR TITLE
feat(bulk-import): add audit log to the backend router [RHIDP-2748]

### PR DESCRIPTION
**Pull request by rm3l**
**What does this PR do / why we need it:**

This PR adds audit log to the bulk-import backend router.

**Which issue(s) this PR fixes:**

Fixes https://issues.redhat.com/browse/RHIDP-2748

**PR acceptance criteria:**

- [ ] Unit tests

- [ ] Integration tests

- [x] Documentation 

**How to test changes / Special notes to the reviewer:**

See https://github.com/rm3l/janus-idp-backstage-plugins/blob/RHIDP-2748--audit-log-bulk-import-backend/plugins/bulk-import-backend/README.md?rgh-link-date=2024-08-27T08%3A06%3A14Z#audit-logging for a complete description of the event names sent by the Bulk Import Backend. The event names are prefixed with `BulkImport`, as now recommended (https://github.com/janus-idp/backstage-plugins/pull/2090).

You can also check the backend logs after sending out a request to any of the Bulk Import Backend API endpoints, e.g.:

```
backend:start: {"actor":{"actorId":"user:default/rm3l","hostname":"localhost","ip":"::1","userAgent":"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/128.0.0.0 Safari/537.36"},"eventName":"BulkImportFindAllImports","isAuditLog":true,"level":"info","message":"'GET /imports' endpoint hit by user:default/rm3l","meta":{},"plugin":"bulk-import","request":{"body":{},"method":"GET","params":{},"query":{"pagePerIntegration":"1","sizePerIntegration":"5"},"url":"/api/bulk-import/imports?pagePerIntegration=1&sizePerIntegration=5"},"response":{"status":200},"service":"backstage","stage":"completion","status":"succeeded","timestamp":"2024-08-27 10:18:18"}

```

### CODE REVIEW

**Review by kim-tsao — COMMENTED** [review]
Is it possible to document the audit log events for eventual inclusion into the product docs?

File: plugins/bulk-import-backend/src/helpers/auditLogUtils.ts
Comment (rm3l) [reply plugins/bulk-import-backend/src/helpers/auditLogUtils.ts:30]: Thanks for the pointers. I'll update that.

Comment (kim-tsao) [reply plugins/bulk-import-backend/src/helpers/auditLogUtils.ts]: it's nice that you were able to incorporate it in the demo today!
/lgtm

**Review by kim-tsao — APPROVED** [review]

**Comment by kim-tsao** [comment]
looks like this plugin has not been added to the codeowners file so prow will not automerge despite satisfying all the necessary approvals.  I will manual merge and update the codeowners file after

**Comment by kim-tsao** [comment]
> looks like this plugin has not been added to the codeowners file so prow will not automerge despite satisfying all the necessary approvals. I will manual merge and update the codeowners file after

nvm, we are covered by the codeowners file.  Maybe we just had to wait a bit

**Comment by rm3l** [comment]
> > looks like this plugin has not been added to the codeowners file so prow will not automerge despite satisfying all the necessary approvals. I will manual merge and update the codeowners file after
> 
> nvm, we are covered by the codeowners file. Maybe we just had to wait a bit

yeah, maybe.. The other PRs that I created in the past related to this plugin were auto-merged by Prow after being approved.
Anyway, thanks for merging it manually.

**Pull request merged**
